packages/liquidator/test/Liquidator.js

```diff
@@ -31,9 +31,9 @@ const { Liquidator } = require("../src/liquidator.js");
 // 2) non-matching 8 collateral & 18 synthetic for legacy UMA synthetics.
```

Comment:
just cleaned up some of these tests to be a bit more consistent. the block below was incorrectly commented out on a previous PR.